Documentation/Signoffs/Resistance_System_Control.md

```diff
@@ -0,0 +1,127 @@
+Team 4 – Mario Kart & Ride Replay Simulator
+
+Team Members: Ray Durlin, Blake Pickett, Tyler Chittum, Benjamin Reed, and Sage Mooneyham
+
+## Detail Design: Resistance System Control Circuit ##
+
+Please note: the scope of this signoff extends only to the control cicuit itself, not to housing components or connectors. Also, the signoff document for the resistance system contains BOM listings and information redundant to this document.
+
+### Function of the Subsystem:
+
+- This subsytem acts to control the linear actuator used in the ride resistance system.
+- Controls both the direction and speed of the linear actuator.
+- Takes an input from the ride replay subsystem over a wireless connection, which determines the distance set point for the linear actuator.
+- Takes an input from the linear actuator feedback to determine the current position of the actuator.
+
+### Constraints:
+
+#### Actuator Duty Cycle
+
+The duty cycle of the utilized linear actuator is 25 %. This value must not be exeeded during normal operation.
+
+#### Actuator Driving Voltage
+
+The utilized actuator is rated for operation at 12 VDC. The control circuit must be capable of providing this volatage to the actuator.
+
+#### Current Capability
+
+The utilized actuator draws a stall current of 7 A, and has a nominal current draw from 1.1 A to 3.4 A. The control circuit must be capable of providing this current to the linear actuator without undergoing thermal overload. In addition, the utilized power connector must be capable of handling the total current demand.
+
+#### Wireless Communications
+
+The control ciruit must be capable of recieving a wirelessly-transmitted digital signal using Bluetooth.
+
+#### Back-EMF Protection
+
+In the course of operation, particularly when the motor comes to a halt after running, a transient voltage potential known as back-EMF will be generated by the motor with opposite polarity to its driving voltage. Due to this, the control cicuit must not be vulnerable to damage from transient reverse-polarization.
+
+#### Mountability
+
+The PCB layout must account for the need to mount the circuit within its housing.
+
+#### Stability
+
+The device must operate without becoming unstable.
+
+#### Resolution
+
+The device must be capable of providing at least 85 unique states, or lengths of extension.
+
+### Schematic and PCB Layout
+
+![Sc_Draft_3](https://user-images.githubusercontent.com/118228609/206243843-0c1c4a77-45d9-469f-8121-5443c982b23b.png)
```

Comment:
Also, I don't think it makes sense to have separate schematics for the transceiver in this case.

Reply:
The transceiver is a pre-made part; therefore, I did not assign it a schematic, as that is not public information to my knowledge. The labels correspond to pins on this pre-made device. If desired, I can draw a box to represent that device, and run the connections to that box, but I do not foresee making a schematic for it.

Reply:
I have a 3D model for the PCB. Would a 3D model which includes the HC06 wireless transceiver suffice as a unified "schematic" for this purpose? I can always add that.

Reply:
Resolved as discussed in class on Wednesday.